_specs/metanorma-model-csa.adoc

```diff
@@ -1,10 +1,10 @@
 ---
 title: Metanorma CSA Normal Document Models
 description: Data models for Metanorma CSA.
-source_url: https://github.com/metanorma/metanorma-model-csand
+source_url: https://github.com/metanorma/metanorma-model-csa
```

Comment:
@ronaldtse make sense to rename https://github.com/metanorma/metanorma-model-csand to https://github.com/metanorma/metanorma-model-csa too

Reply:
@CAMOBAP795 yes, done.